_overviews/core/futures.md

```diff
@@ -844,8 +823,9 @@ by the clients-- they can only be called by the execution context.
 When asynchronous computations throw unhandled exceptions, futures
 associated with those computations fail. Failed futures store an
 instance of `Throwable` instead of the result value. `Future`s provide
-the `onFailure` callback method, which accepts a `PartialFunction` to
-be applied to a `Throwable`. The following special exceptions are
+the `failed` projection method, which allows this `Throwable` to be
+treated as the success value of another `Future`.
+The following special exceptions are
```

Comment:
I guess you were trying to avoid touching the next line unnecessarily but it's made the hard-wrapping a little untidy.